lib/Client.php

```diff
@@ -134,7 +132,7 @@ public function transaction() {
      */
     public function close() {
         /** @var Promise $promise */
-        $promise = all(promises($this->promisors));
+        $promise = all($this->promisors);
```

Comment:
I think this should be rather a static function on `Deferred`, as, unlike all our other functions, does not operate on the generic primitives, but on Deferred itself. I.e. `Deferred::promises(array $deferreds)`. Also is more obvious IMHO. Had to think for like 15 seconds have exactly `promises()` should do.